terraform/prod.tfvars

```diff
@@ -52,6 +52,9 @@ tenant_cidrs = [
   "194.73.212.3/32",
   # MOJ Digital and Technology
   "81.134.202.29/32",
+  # UK Cloud / Skyscape egress for #140308141
+  185.40.8.212/32,
+  86.188.177.234/32,
```

Comment:
Bit of an odd situation, but there's some context in the referenced story: https://www.pivotaltracker.com/n/projects/1275640/stories/140308141